types/node/v4/node-tests.ts

```diff
@@ -363,6 +363,11 @@ function stream_readable_pipe_test() {
     r.close();
 }
 
+function gzipText() {
```

Comment:
1. All tests are separated by modules. This test related to `zlib` module and for consistency should have comment 
  
   ```
   ////////////////////////////////////////////////////
   /// zlib tests : http://nodejs.org/api/zlib.html ///
   ////////////////////////////////////////////////////
   ```

2. Also for consistency maybe better use `namespace` instead function:

   ```ts
   namespace zlib_tests {
       const gzipped = zlib.gzipSync('test');
       const unzipped = zlib.gunzipSync(gzipped);
   }
   ```